testdata/terraform/cloudflare_managed_headers/test.tf

```diff
@@ -0,0 +1,23 @@
+resource "cloudflare_managed_headers" "terraform_managed_resource" {
+  zone_id = "0da42c8d2132a9ddaf714f9e7c920711"
+  managed_request_headers {
+    enabled = true
+    id      = "add_visitor_location_headers"
+  }
+  managed_request_headers {
```

Comment:
i wouldn't expect the `enabled = false` ones to be outputted in TF.

Reply:
ah right, i update the code, just not the test, let me fix that.